test/vanilla/Expected/AcceptanceTests/Constants/constants/models/_models.py

```diff
@@ -164,47 +164,33 @@ def __init__(self, **kwargs):
 class NoModelAsStringNoRequiredOneValueDefault(msrest.serialization.Model):
     """NoModelAsStringNoRequiredOneValueDefault.
 
-    Variables are only populated by the server, and will be ignored when sending a request.
-
-    :ivar parameter:  Default value: "value1".
-    :vartype parameter: str
+    :param parameter:
+    :type parameter: str
     """
 
-    _validation = {
-        "parameter": {"constant": True},
-    }
-
     _attribute_map = {
         "parameter": {"key": "parameter", "type": "str"},
     }
 
-    parameter = "value1"
-
     def __init__(self, **kwargs):
         super(NoModelAsStringNoRequiredOneValueDefault, self).__init__(**kwargs)
+        self.parameter = kwargs.get("parameter", "value1")
 
 
 class NoModelAsStringNoRequiredOneValueNoDefault(msrest.serialization.Model):
     """NoModelAsStringNoRequiredOneValueNoDefault.
 
-    Variables are only populated by the server, and will be ignored when sending a request.
-
-    :ivar parameter:  Default value: "value1".
-    :vartype parameter: str
+    :param parameter:
```

Comment:
[swagger](https://github.com/Azure/autorest.testserver/blob/main/swagger/constants.json#L86). This is an optional constant with no default, so we default to `None`.

The doc should also be improved here, so we show the one non-`None` value. This is included in the [same docstring issue](https://github.com/Azure/autorest.python/issues/953)

Reply:
add docstring fix to this PR